uvahelpr_web/marketplace/views.py

```diff
@@ -58,4 +58,37 @@ def allJobs(request):
 	return render(request, 'marketplace/jobslist.html', context)
 
 
+def create_account(request):
+	if request.method == 'GET':
+		form = CreateAccountForm()
+		return render(request, 'marketplace/create_account.html', {'form': form})
+	form = CreateAccountForm(request.POST)
+	# check whether it's valid:
+	if not form.is_valid():
+	 	return render(request, 'marketplace/create_account.html', {'form': form})
+	username = form.cleaned_data['username']
+	email = form.cleaned_data['email']
+	password = form.cleaned_data['password']
+	first_name = form.cleaned_data['first_name']
+	last_name = form.cleaned_data['last_name']
+	phone_number = form.cleaned_data['phone_number']
+	skills = form.cleaned_data['skills']
+	post_data = {'username': username,
+				 'email': email,
+				 'password': password,
+				 'first_name':first_name,
+				 'last_name':last_name,
+				 'phone_number': phone_number,
+				 'skills': skills}
+	post_encoded = urllib.parse.urlencode(post_data).encode('utf-8')
+	req = urllib.request.Request('http://exp-api:8000/create_account/', data=post_encoded, method='POST')
+	resp_json = urllib.request.urlopen(req).read().decode('utf-8')
+	resp = json.loads(resp_json)
+	if not resp or not resp['ok']:
+	 	# couldn't create account, send them back to account page with error
+	 	return render(request, 'marketplace/create_account.html', {'form': form, 'error':True})
+	 # created account, redirect to success page
+	return HttpResponseRedirect(reverse('create_account_success'))
```

Comment:
We should log in the user after successful creation and redirect to the index page instead of a separate 'account successful' page